src/main/java/dev/jbang/cli/Run.java

```diff
@@ -185,8 +184,8 @@ String generateCommandLine(Script script) throws IOException {
 					optionalArgs.add(classpath);
 				}
 
-				if (optionActive(cds(), script.enableCDS())) {
-					String cdsJsa = script.getJar().getAbsolutePath() + ".jsa";
+				if (xrunit.runUnit instanceof Script && optionActive(cds(), xrunit.script().enableCDS())) {
```

Comment:
I don't grok why this should only apply for Scripts and not Jars ?

Reply:
Because the original code checks for the existence of `//CDS` which can only happen in scripts? Or did I understand this wrong? (But then I don't understand how it works)

Reply:
I think the old code was wrong :)

CDs is applicable to any launch Afaics. (Except native)

Reply:
Ok, I can certainly add that.

Btw, how do you force CDS *off* if you don't want to use it?